jupyter_scheduler/environments.py

```diff
@@ -37,10 +44,7 @@ def list_environments(self) -> List[RuntimeEnvironment]:
                     label=name,
                     description=f"Conda environment: {name}",
                     file_extensions=["ipynb", "py"],
-                    output_formats=[
-                        OutputFormat(name="ipynb", label="Notebook"),
-                        OutputFormat(name="html", label="HTML"),
-                    ],
+                    output_formats=["ipynb", "html"],
```

Comment:
Longer term, we will probably need to change how this works, to allow a different set of output formats for each file extension that the environment supports. For example, a python script might have stdout and stderr as output formats, but not notebooks. We don't need to deal with that now though.